ansible/roles/ocp4-workload-open-data-hub-olm/defaults/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+become_override: False
+ocp_username: opentlc-mgr
+silent: False
+
+odh_namespace: "opendatahub"
```

Comment:
Please change variable names (except the first three) to use the `ocp4_workload_open_data_hub_olm_` prefix. For example `ocp4_workload_open_data_hub_olm_namespace`